### DISCUSSION TITLE
SSO integration

### DISCUSSION THREAD

**Discussion by 0xpr03** [F43nd1r/Acrarium#340] [Ideas]
On my current wish list something working with SAML / OAUTH (keycloak) for SSO integration would be a really nice to have. How hard would this be to accomplish ?

**Comment by F43nd1r** [comment]
I don't really have time for the project currently. There won't be any new features unless someone else implements them, regardless of how hard they might be.
SSO should not be too hard to integrate, there's probably tutorials on how to do that in spring and then the only thing left is the UI.
